package.json

```diff
@@ -11,25 +11,25 @@
   "homepage": ".",
   "license": "MIT",
   "dependencies": {
-    "@graasp/apps-query-client": "github:graasp/graasp-apps-query-client.git",
-    "@graasp/ui": "git://github.com/graasp/graasp-ui.git",
-    "@material-ui/core": "4.12.3",
-    "@material-ui/icons": "4.11.2",
-    "@material-ui/lab": "4.0.0-alpha.60",
-    "@uppy/core": "2.1.4",
-    "@uppy/dashboard": "2.1.3",
-    "@uppy/drag-drop": "2.0.5",
+    "@graasp/apps-query-client": "github:graasp/graasp-apps-query-client.git#22/fetchFile",
```

Comment:
Now that the branch has been merged, you can change remove the explicit branch specifier.